### DISCUSSION TITLE
Make card darkmode even if in lightmode

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [collab-community/journey-book#117] [General]
I was trying to make the darkmode color actually black instead of dark blue, but I messed something up so the cards became in the dark blue color, it actually looks nice even in lightmode, so if @collab-community/journey-book likes this, can we make the user card perm this color?

![image](https://user-images.githubusercontent.com/84712013/184140323-196dd220-68fd-4906-ae6f-625242dcf69c.png)

**Comment by agr1ms** [comment]
I think, all the other texts(heading and nav items) are very light.
Rest, nothing wrong with this dark blue color
Creating an optical illusion though...
